kanidmd/idm/src/constants/schema.rs

```diff
@@ -214,6 +214,38 @@ pub const JSON_SCHEMA_ATTR_DOMAIN_NAME: &str = r#"{
       ]
     }
 }"#;
+
+pub const JSON_SCHEMA_ATTR_DOMAIN_DISPLAY_NAME: &str = r#"{
+  "attrs": {
+    "class": [
+      "object",
+      "system",
+      "attributetype"
+    ],
+    "description": [
+      "The user-facing display name of the Kanidm domain."
+    ],
+    "index": [
+      "EQUALITY",
+      "PRESENCE"
```

Comment:
I'm not sure you need presence here.

Reply:
I may just be yeeting random things around while I figure out how to modify schema 🙄